netstandard/ref/System.Globalization.cs

```diff
@@ -203,7 +204,7 @@ public partial class CultureInfo : System.ICloneable, System.IFormatProvider
         public static System.Globalization.CultureInfo ReadOnly(System.Globalization.CultureInfo ci) { throw null; }
         public override string ToString() { throw null; }
     }
-    public partial class CultureNotFoundException : System.ArgumentException, System.Runtime.Serialization.ISerializable
+    public partial class CultureNotFoundException : System.ArgumentException
```

Comment:
Why is `ISerializable` being removed?

Reply:
Because it's now [directly on Exception](https://github.com/dotnet/standard/blob/master/netstandard/ref/System.cs#L2156).

Reply:
Cool, so I consider @dotnet/nsboard-foundation to have signed off then.